### PR TITLE
community: make bing web search as the only option

### PR DESCRIPTION
**Pull request by mainred**
This PR make bing web search as the option for BingSearchAPIWrapper to facilitate and simply  the user interface on Langchain.
This is a follow-up work of https://github.com/langchain-ai/langchain/pull/23306.

### CODE REVIEW

**Comment by mainred** [comment]
@ccurme could you please take a look?

File: libs/community/langchain_community/utilities/bing_search.py [hunk 1]
```diff
@@ -5,16 +5,25 @@
 from langchain_core.pydantic_v1 import BaseModel, Extra, Field, root_validator
 from langchain_core.utils import get_from_dict_or_env
 
-# BING_SEARCH_ENDPOINT is the default endpoint for Bing Search API and is normally
-# invariant to users.
+# BING_SEARCH_ENDPOINT is the endpoint for Bing Web Search API.
+# Because Bing Search is a service provided by Microsoft, you need to
```
Comment (ccurme) [inline libs/community/langchain_community/utilities/bing_search.py:9]: typo / incomplete sentence

**Comment by ccurme** [comment]
@mainred what is the motivation for removing users' ability to customize the URL? Having the default at `BING_SEARCH_ENDPOINT` seems sensible to me. Are users using this feature now?

**Comment by mainred** [comment]
> @mainred what is the motivation for removing users' ability to customize the URL? Having the default at BING_SEARCH_ENDPOINT seems sensible to me. 
I reverted my code and the user can still customized their endpoint. Added more context of Bing Search Service to allow people understand the background.

> Are users using this feature now?
Bing Search API s is active as a separate service since 31 October 2023.  
ref: https://azure.microsoft.com/en-in/updates/bing-search-apis-will-transition-from-azure-cognitive-services-to-azure-marketplace-on-31-october-2023/

**Review by ccurme — APPROVED** [review]

**Pull request merged**